### ISSUE TITLE
doctrine:mapping:import fails on Symfony flex

### ISSUE THREAD

**Issue by Steveb-p** [doctrine/DoctrineBundle#815]
I was following this documentation: https://symfony.com/doc/current/doctrine/reverse_engineering.html

When calling `doctrine:mapping:import` on a new Symfony flex application it fails due to absence of usual AppBundle.

Since from what I gather, the only reason bundle was required in the first place was to establish destination location for created files, would it be ok if it was removed and replaced with an argument to pass destination directly?

Code in question below extracted from `Doctrine\Bundle\DoctrineBundle\Command\ImportMappingDoctrineCommand` line 77.
```php
$bundle = $this->getApplication()->getKernel()->getBundle($input->getArgument('bundle'));

$destPath = $bundle->getPath();
$type = $input->getArgument('mapping-type') ? $input->getArgument('mapping-type') : 'xml';
if ('annotation' === $type) {
    $destPath .= '/Entity';
} else {
     $destPath .= '/Resources/config/doctrine';
}
```

`$bundle` seems to not be used afterwards.

**Comment by weaverryan** [comment]
Hey!

Are you using the latest version of this bundle? We actually already updated that command to accept a namespace instead of a bundle (and that’s reflected in the docs). A bundle is still allowed, but in the latest version, you should be able to pass a namespace without any issues.

Cheers!

**Comment by Steveb-p** [comment]
@weaverryan indeed the project I was working on had 1.8.1 locked. I see there was a patch that landed on 1.9.0 which resolves this issue.

Sorry and thank you very much :)
